docs/source/conf.py

```diff
@@ -0,0 +1,37 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+import sys
+
+sys.path.insert(0, "src")
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = "rashdf"
+copyright = "2024, fema-ffrd"
+author = "fema-ffrd"
+release = "0.2.1"
+
+# -- General configuration ---------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
+
+extensions = [
+    "sphinx.ext.autodoc",
+    "sphinx.ext.autosummary",
+    "sphinx_rtd_theme",
+    "numpydoc",
+]
+
+templates_path = ["_templates"]
+exclude_patterns = []
+
```

Comment:
I was getting the following warning when building the docs:
`WARNING: autosummary: stub file not found 'rashdf.RasGeomHdf.bc_lines'. Check your autosummary_generate setting.`

I found [this thread](https://stackoverflow.com/questions/65198998/sphinx-warning-autosummary-stub-file-not-found-for-the-methods-of-the-class-c) and added the following line to squash it:
```suggestion
numpydoc_show_class_members = False
```